.travis.yml

```diff
@@ -1,8 +1,18 @@
 language: go
+
+services:
+  - docker
+
 go:
   - 1.7
   - 1.8
 install:
   -
 script:
   - ./test
+  - ./docker-build
+after_success:
```

Comment:
You can do this in a better way: https://github.com/coreos/matchbox/blob/master/.travis.yml#L18

Reply:
Mostly a nit, doing `"$TRAVIS_BRANCH" == "master"` isn't the worst

Reply:
Seems to have used what travis documents https://docs.travis-ci.com/user/docker/#Branch-Based-Registry-Pushes, but the matchbox way looks cleaner